### PR TITLE
Lazy optimizations

### PR DESCRIPTION
**Pull request by braaannigan**

### CODE REVIEW

**Comment by braaannigan** [comment]
@ritchie46 It would be good if you could have a look at this and let me know if it's on the right track. Still in draft though

File: user_guide/src/lazy-api/lazy-query-plan.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+### Printed query plan
+
+We can also print the non-optimized plan with `describe_plan`
+
+```python
+{{#include ../examples/lazy_api/snippet1.py:11:11}}
+```
+
+```text
+{{#include ../outputs/lazy_api/q1_plan.txt}}
```
Comment (ritchie46) [inline user_guide/src/lazy-api/lazy-query-plan.md:37]: Doesn't render

**Review by ritchie46 — COMMENTED** [review]
Thanks @braaannigan. I have left some remarks, but great addition to book.

**Comment by braaannigan** [comment]
@ritchie46 Made those changes, this is good to go

**Comment by r-brink** [comment]
@braaannigan Some of the output cells are not working https://pola-rs.github.io/polars-book/user-guide/lazy-api/lazy-schema.html

image

**Pull request merged**